localflavor/kw/kw_governorates.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.utils.translation import ugettext_lazy as _
+
+# List of governorates as defined by ISO_3166-2
+# For reference; see https://en.wikipedia.org/wiki/ISO_3166-2:KW
+# Note that these are governorates and not subdivisons
+
+# These are the English names
+GOVERNORATE_CHOICES = (
+    ('AH', _('Ahmadi')),
+    ('FA', _('Farwaniyah')),
+    ('JA', _('Jahra')),
+    ('KU', _('Capital')),
+    ('HA', _('Hawalli')),
+    ('MU', _('Mubarak Al Kabir')),
```

Comment:
I wouldn't mind that - except in Real Life - the KW- is dropped as its redundant.